pom.xml

```diff
@@ -41,22 +41,32 @@
       <tag>HEAD</tag>
   </scm>
 
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>software.amazon.awssdk</groupId>
+        <artifactId>bom</artifactId>
+        <version>2.15.20</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
   <dependencies>
 
     <dependency>
-      <groupId>com.amazonaws</groupId>
```

Comment:
Please leave the dependency style as it was

Reply:
Done : https://github.com/prometheus/cloudwatch_exporter/pull/307/commits/694c71e035c532588d534dacb4f4c5fe9c3a2c4e